examples/mixer_demo.rs

```diff
@@ -73,22 +80,30 @@ fn demo(music_file: &Path, sound_file: &Path) {
     println!("music volume => {:?}", sdl2::mixer::Music::get_volume());
     println!("play => {:?}", music.play(1));
 
-    let sound_chunk_res = sdl2::mixer::Chunk::from_file(sound_file);
-
-    match sound_chunk_res {
-        Ok(sound_chunk) => {
-            println!("chunk volume => {:?}", sound_chunk.get_volume());
-            println!("playing sound twice");
-            let play_res = sdl2::mixer::Channel::all().play(&sound_chunk, 1);
-
-            timer.delay(5000);
-
-            match play_res {
-                Ok(_) => { println!("played sound")},
-                Err(e) => { println!("{:?}", e) },
+    match sound_file {
```

Comment:
`if let Some(sound_file_path) = sound_file {` and get rid of the `None` branch ;)